student.rb

```diff
@@ -1,15 +1,10 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'school'
+
+class Student < School
```

Comment:
A student inherits from a school? those are not similar types of objects. A school is a building. A student is a human. What attributes or state would they share?